subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.c

```diff
@@ -117,6 +114,15 @@ int lll_adv_init(void)
 {
 	int err;
 
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
+#if defined(CONFIG_BT_CTLR_ADV_AUX_SET)
```

Comment:
Nitpick: `#if defined(CONFIG_BT_CTLR_ADV_EXT)` isn't really necessary here

Reply:
```suggestion
#if defined(CONFIG_BT_CTLR_ADV_EXT)
#if (BT_CTLR_ADV_AUX_SET > 0)
```

note, without `CONFIG_`